rest-api/src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
 spring.main.banner-mode=off
 spring.http.multipart.max-file-size=-1
 spring.http.multipart.max-request-size=-1
+logging.level.org.springframework.boot.autoconfigure.security.AuthenticationManagerConfiguration=OFF
```

Comment:
Once @clydet's change to stop the password from logging goes in, we will remove this.